justfile

```diff
@@ -237,6 +235,9 @@ ipython:
     # The resulting output will be at `api/static` and is git ignored for convenience.
     @STATIC_ROOT="./static" just dj collectstatic --noinput
 
+@collectstatic-as-admin: _api-up
+    just exec -u root web python manage.py collectstatic --noinput
+
```

Comment:
What is the use-case for this recipe? Is it left-over from debugging, maybe?

For good measure I tried out the nginx build flow locally on this branch and it worked fine, as far as I can tell. The files get copied into the nginx image and are accessible. Thank you for making sure that still works :rocket:

Reply:
Indeed it's a leftover. Deleted, thanks!